src/selectors/layout.js

```diff
@@ -149,14 +66,18 @@ export const getChartSize = createSelector(
   }
 );
 
+// Check that width & height props are present and finite
+const isFinite = n => typeof n !== 'undefined' && Number.isFinite(n);
+const isValid = d => isFinite(d.width) && isFinite(d.height);
```

Comment:
I'm not sure that I agree, because the point of the function is just to check that the input has valid values, and you don't need to know how the function works when reading the name, just to know what it's supposed to accomplish.

Reply:
Yeah I see what you mean and here it's definitely not a big deal since the function is defined locally, so I won't press on it. But in case you're interested in my reasoning, I meant based on its usage:

`if (!isValid(graph) || !isValid(chart))`

In some ways suggests `isValid` could have a stronger definition than is needed for `getZoomPosition`?

If that's the intention the name is fine, but then e.g. `getZoomPosition` becomes overly constrained if the `isValid` implementation gets stricter for some other usage. If that's not what you'd want, then often I think a name that describes what the function does ends up clearer.